src/java/org/apache/cassandra/tools/nodetool/layout/CassandraCliHelpLayout.java

```diff
@@ -0,0 +1,786 @@
+    public static final String SYNOPSIS_SUBCOMMANDS_LABEL = "<command> [<args>]";
+    public static final String SUBCOMMAND_OPTION_TEMPLATE = "With %s option, %s";
+    public static final String SUBCOMMAND_SUBHEADER = "With no arguments, Display help information";
+    private static final String[] EMPTY_FOOTER = new String[0];
+
+    public CassandraCliHelpLayout(CommandLine.Model.CommandSpec spec, ColorScheme scheme)
+    {
+        super(spec, scheme);
+    }
+
+    @Override
+    public String descriptionHeading(Object... params)
+    {
+        return createHeading(DESCRIPTION_HEADING, params);
+    }
+
+    /**
+     * @param params Arguments referenced by the format specifiers in the header strings
+     * @return the header string.
+     */
+    @Override
+    public String description(Object... params)
+    {
+        CommandLine.Model.CommandSpec spec = commandSpec();
+        String fullName = spec.qualifiedName();
+
+        TextTable table = TextTable.forColumns(colorScheme(),
+                                               new Column(spec.usageMessage().width() - COLUMN_INDENT, COLUMN_INDENT,
+                                                          Column.Overflow.WRAP));
+        table.setAdjustLineBreaksForWideCJKCharacters(spec.usageMessage().adjustLineBreaksForWideCJKCharacters());
+        table.indentWrappedLines = 0;
+
+        table.addRowValues(colorScheme().commandText(fullName)
+                                        .concat(" - ")
+                                        .concat(colorScheme().text(String.join(" ", spec.usageMessage().description()))));
+        table.addRowValues(Ansi.OFF.new Text("", colorScheme()));
+        return table.toString(new StringBuilder()).toString();
+    }
+
+    @Override
+    public String synopsisHeading(Object... params)
+    {
+        return createHeading(SYNOPSIS_HEADING, params);
+    }
+
+    /**
+     * This method is overridden to provide a detailed synopsis for the command and its subcommands.
+     * <pre>
+     * {@code
+     *     SYNOPSIS
+     *         nodetool [(-h <host> | --host <host>)] [(-p <port> | --port <port>)]
+     *                 [(-pp | --print-port)] [(-pw <password> | --password <password>)]
+     *                 [(-pwf <passwordFilePath> | --password-file <passwordFilePath>)]
+     *                 [(-u <username> | --username <username>)] bootstrap <command>
+     *                 [<args>]
+     * }
+     * </pre>
+     *
+     * @param synopsisHeadingLength the length of the synopsis heading that will be displayed on the same line
+     * @return The synopsis string.
+     */
+    @Override
+    public String synopsis(int synopsisHeadingLength)
+    {
+        return printDetailedSynopsis("", COLUMN_INDENT, true);
+    }
+
+    private String printDetailedSynopsis(String synopsisPrefix, int columnIndent, boolean showEndOfOptionsDelimiter)
```

Comment:
nit: Not necessarily a problem, but these arguments are always `"", COLUMN_INDENT, true` respectively.

Reply:
Fixed.